.github/workflows/package_linux.yml

```diff
@@ -0,0 +1,47 @@
+name: Linux / Package
+
+on:
+  push:
+    branches:
+      - master
+      - 'releases/**'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    container:
+      image: vowpalwabbit/manylinux2010-build:latest
+    runs-on: ubuntu-latest
+    steps:
+      # v1 must be used because newer versions require a node.js version that will not run on this old image.
+      - uses: actions/checkout@v1
+      - name: Configure
+        shell: bash
+        run: |
+          cmake -S . -B build -DSTATIC_LINK_VW_JAVA=ON -DVW_INSTALL=OFF -DVW_INSTALL_EXE=ON -DBUILD_TESTS=OFF -DCMAKE_BUILD_TYPE=Release
+      - name: Build
+        shell: bash
+        run: |
+          cmake --build build --parallel $(nproc) --target all
+      - name: Package tar.gz
+        shell: bash
+        run: |
+          cd build
+          cpack -G TGZ
+      - name: Package Deb
+        shell: bash
+        run: |
+          cd build
+          cpack -G DEB
+      - name: Upload tar.gz
```

Comment:
this could also be used to package the header files and vw library right? or at least in theory

Reply:
mention in the PR desc the musl callout

Reply:
Yes, the problem is that libvw is usually used statically - so you've got lots of compatibility issues there and so doing this in a portable way needs something like nix (which is already done). Shipping the DLL this way would be good but I just omitted here for simplicity. Musl would still not be the whole story I think since I am not sure about binary compat across compilers, so you'd need to match those too (nix to the rescue). Additionally, since the static lib depends on spdlog and fmt those need to be shipped too if you decide to package a static lib (and it just feels odd to do this) - that's the primary reason for the VW_INSTALL_EXE change in this PR.